src/components/ui/FileBrowser/ZarrPreview.tsx

```diff
@@ -136,138 +136,122 @@ export default function ZarrPreview({
                 Open with:
               </Typography>
               <ButtonGroup className="relative">
-                <Tooltip placement="top">
-                  <Tooltip.Trigger
-                    as={Button}
-                    variant="ghost"
-                    className="rounded-sm m-0 p-0 transform active:scale-90 transition-transform duration-75"
-                  >
-                    <Link
-                      to={openWithToolUrls.validator}
-                      target="_blank"
-                      rel="noopener noreferrer"
+                {openWithToolUrls.validator ? (
```

Comment:
Placing each one of these in a component would make this code easier to read and possibly the component could be parameterized, in order to reduce the tooltip boiler plate.